.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint-and-build:
+    name: "🔍 Lint, Type Check & Build"
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ "22.x" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)